101-backup-la-reporting/azuredeploy.json

```diff
@@ -0,0 +1,176 @@
+{
+    "$schema":"https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion":"1.0.0.0",
+    "parameters":{
+      "workspaceLocation": {
```

Comment:
rename this to "location" - add a defaultValue of [resourceGroup().location]

Reply:
Updated